### DISCUSSION TITLE
Simulate "Not Published" status in preview environment

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#897] [Q&A] [answered]
Is there a way to obtain/simulate the "Not Published" status on a notice in the Preview environment?

Thanks in advance,
Alex

**Comment by gennidou** [comment, marked as answer]
Hi Alex
As stated in our [Preview](https://docs.ted.europa.eu/eforms-common/preview/index.html#_enotices2:~:text=Precondition%20for%20this%20is%20that%20the%20notice%20triggers%20lawfulness%20warning%20and%20we%20receive%20your%20request%20to%20reject%20it%20by%20business%20ID%20(i.e.%20notice%20ID%20%2B%20version%20ID).) page, precondition for this is that the notice triggers lawfulness warning and we receive your request to reject it by business ID (i.e. notice ID + version ID).
Please first make sure that your notice has [lawfulness](https://docs.ted.europa.eu/eforms/latest/reference/business-rules/index.html#_lawfulness) warning (and no errors), which you can check in CVS before submitting to eNotices2 Preview. This request should be done with Helpdesk, but if it's urgent you can exceptionally provide me with the BusinessID (notice ID + version ID) here once you have submitted, so that I can reject it.
Thanks
